src/app/pages/onboarding/sign-in/mnemonic-form.tsx

```diff
@@ -100,14 +97,16 @@ export function MnemonicForm({ mnemonic, setMnemonic, twentyFourWordMode }: Mnem
                 aria-busy={isLoading}
                 width="100%"
                 type="submit"
+                // need to check the disabled variants of solid button
```

Comment:
What does this mean?

Reply:
This relates to when I was having those issues with button styles, I think it's OK to remove now. Thanks!